composer.json

```diff
@@ -59,8 +59,8 @@
         "symfony/process": "3.4.2"
     },
     "require-dev": {
-        "helmich/phpunit-json-assert": "^2.1 || ^3.0",
-        "phpunit/phpunit": "^8"
+        "helmich/phpunit-json-assert": "^3.0",
+        "phpunit/phpunit": "^8.2.4"
```

Comment:
👍 

I understand developers have different opinions on this, but personally, I always use full version constraints (i.e., `major.minor.patch`). In my opinion, this documents awareness of features and bug fixes well.